src/Components/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,47 @@
+import styles from "./SideBar.module.scss";
+
+const SideBar = () => {
+  const tags = [
+    "Kotlin",
+    "Vue Js",
+    "Android",
+    "ReactJs",
+    "Node",
+    "Placements",
+    "Embedded Structures",
+    "Data Structures",
+    "react",
+    "vueJs",
+    "Kotlin",
+    "pnpm",
+    "Vite",
+    "scss",
+  ];
+
+  const tagspan = tags.map((tag) => <span>{tag}</span>);
```

Comment:
Even though this is technically allowed, try to always use JSX inside the return statement (where JSX is supposed to be), as this is prone to recalculating the map.

Place this directly inside JSX.

Also, always use a key with a map. A [good reference from docs](https://reactjs.org/docs/lists-and-keys.html)